### PR TITLE
created a common header

### PR DESCRIPTION
**Pull request by vricci518**
Created a common header for Aboutme site

### CODE REVIEW

**Comment by vricci518** [comment]
I've created a common header file to be used on all of the pages. That header shows up on every page, but the dropdown menu for the Links does not show. When viewing just the header.php that dropdown is visible.

File: new_aboutme/about_me2/header.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Bootstrap CSS -->
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+	<link rel="stylesheet" href="css/aboutme2.css">
+
+	<title></title>
+</head>
```
Comment (billythakidd04) [inline new_aboutme/about_me2/header.php]: None of this is needed when just including a snippet. When you do the `inlcude()` function it is essentially dropping the  contents of that file right in to the other one. So if you think about where it is included, you now have an entire HTML doc inside of the original one.

File: new_aboutme/about_me2/header.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+					</ul>
+				</div>
+			</div>
+		</nav><br />
+	</header>
+	<!-- Option 1: Bootstrap Bundle with Popper -->
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+</body>
+
+</html>
```
Comment (billythakidd04) [inline new_aboutme/about_me2/header.php]: Don't need any of this either

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
See comment about header.php

File: new_aboutme/about_me2/header.php (hunk 2 above)
Comment (vricci518) [reply new_aboutme/about_me2/header.php]: adjusted the header.php file and dropdown menu shows up now

**Comment by vricci518** [comment]
Closed #3

**Pull request merged**